instrumentation/log4j/log4j-appender-2.17/library/src/main/java/io/opentelemetry/instrumentation/log4j/appender/v2_17/OpenTelemetryAppender.java

```diff
@@ -139,14 +152,23 @@ private static List<String> splitAndFilterBlanksAndNulls(String value) {
         .collect(Collectors.toList());
   }
 
+  public void setOpenTelemetry(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  public OpenTelemetry getOpenTelemetry() {
```

Comment:
Shouldn't it be private?
```suggestion
  private OpenTelemetry getOpenTelemetry() {
```